draft-packages/form/KaizenDraft/Form/TextAreaField/TextAreaField.spec.tsx

```diff
@@ -30,3 +30,45 @@ it("renders a validation message when provided", () => {
 
   expect(screen.getByText("Incorrect message")).toBeTruthy()
 })
+
+it("renders a prominent label", () => {
+  const { container } = render(
+    <TextAreaField
+      id="reply"
+      labelText="Prominent label"
+      validationMessage="Incorrect message"
+      variant="prominent"
+    />
+  )
+
+  expect(container.getElementsByClassName("prominent").length).toBe(1)
+})
+
+it("renders descriptions as the first sibling of the prominent labels", () => {
+  const { container } = render(
+    <TextAreaField
+      id="reply"
+      labelText="Prominent label"
+      description="This should be the immediate sibling to the label"
+      variant="prominent"
+    />
+  )
+
+  const prominentLabel = container.getElementsByClassName("prominent")[0]
+  const siblingNode = prominentLabel?.nextElementSibling
+
+  expect(siblingNode?.className === "message").toBeTruthy()
+})
+
+it("Does not render a validation message if it is disabled", () => {
```

Comment:
When there is an `if` in a test,  it usually means there should be 2 tests at least. You are showing that it won't show the validation message when it is `disabled`, but for all I know it never shows it even when it is "enabled".
It might look verbose, but I'd so something like this here:
```
describe("Validation message", () => {
  describe("when the textarea field is NOT disabled", () => { // I like capitals here because it stands out against the other describe block
    it("renders the validation message", ...)
  })

  describe("when the textarea field is disabled", () => {
    it("does NOT render the validation message", ...)
  })
})
```
This also make it look quite nice in the output of the testrunner.

This structure is also really helpful when you have multiple tests for each `when...` block as you can have one `beforeEach` where you set up the state for all.

Reply:
True, good opportunity to test for happy and sad path :) I've updated this now to account for both test cases